content/post/2017-09-03-psychology-in-crisis.md

```diff
@@ -0,0 +1,397 @@
+
+# 科学危机
+
+GVT只不过是这几年科学可重复性危机的冰山一角，在心理学领域大量重复失败的研究里[<sup>11</sup>](#open2015estimating)，GVT的论文质量算是比较好的了。我再随便举两个出了问题的研究：
+
+*   哈佛美女教授Amy Cuddy在TED上的[著名演讲](https://www.ted.com/talks/amy_cuddy_your_body_language_shapes_who_you_are?language=zh-cn)，认为强势的身体动作(power pose)可以助你成功。这个演讲在TED上观看次数超过四千两百万，是历史第二多，当时在社交网站上也掀起了一波"Fake it until you make it"的鸡汤潮。结果不仅没重复出来，原论文的另一个作者还直接在网上写了一封[公开信](http://faculty.haas.berkeley.edu/dana_carney/pdf_My%20position%20on%20power%20poses.pdf)，声称观点已变，现在认为power pose效应不存在了，建议其他研究者别再浪费时间。![Amy Cuddy在TED的演讲](https://raw.githubusercontent.com/tcya/tcya.github.io/master/assets/images/power_pose.png)
+*   社会启动效应(social priming)。这个领域发端于John Bargh1996年的研究[<sup>12</sup>](#bargh1996automaticity)，他们让学生从五个词里挑四个出来造句，一组学生拿到的是普通词，另一组拿到的是与老年相关的词汇，比如健忘、秃顶、满脸皱纹、佛罗里达，实验完后他们告诉学生穿过大厅去另一个地方做下一组实验，但他们关心的其实既不是造句也不是下一个实验，而是学生穿过走廊的速度，他们发现那些被老年相关词汇启动(prime)了的学生比另一组慢了将近一秒。这个惊人的结果启发了大量后续研究，十几年来整个领域欣欣向荣，Bargh的原论文已被引用了四千多次。人们发现啥都能拿来prime，比如让你描述一个严厉教授的样子你考试就能[考得更好](http://blogs.discovermagazine.com/neuroskeptic/2016/10/13/social-priming-works-after-all/#.WW2hzdPyvdQ)，考卷字迹模糊也能让你[考得更好](http://weibo.com/p/1001603935981584676451)……卡尼曼在《思考：快与慢》里举了整整一章的例子，我们来稍微感受一下， ![《思考：快与慢》中关于社会启动效应的论述1](https://raw.githubusercontent.com/tcya/tcya.github.io/master/assets/images/social_priming_1.png) ![《思考：快与慢》中关于社会启动效应的论述2](https://raw.githubusercontent.com/tcya/tcya.github.io/master/assets/images/social_priming_2.png) ![《思考：快与慢》中关于社会启动效应的论述3](https://raw.githubusercontent.com/tcya/tcya.github.io/master/assets/images/social_priming_3.png) 请注意卡尼曼最后的结论，“你别无选择”、“你必须承认”……好吧，现在Bargh的最初实验和其他许多实验都[没能重复出来](http://www.guokr.com/article/436704/)，整个领域风雨飘摇。Bargh前几年没憋住还[发了通火](http://blogs.discovermagazine.com/notrocketscience/2012/03/10/failed-replication-bargh-psychology-study-doyen/)，又引来更多非议。这里有[一篇](https://rolfzwaan.blogspot.com/2013/09/how-to-cook-up-your-own-social-priming.html)手把手教你做social priming研究的文章，辛辣搞笑，推荐阅读。当然我不是说这领域一定就全错了，毕竟我也不是专业人士，我只是觉得怎么看我应该都还是[有点选择](http://andrewgelman.com/2014/09/03/disagree-alan-turing-daniel-kahneman-regarding-strength-statistical-evidence/)的。
+
+所以心理学到底是怎么落到这步田地的呢？
+
+## 统计困局
+
+> There are three kinds of lies: lies, damned lies, and statistics. - Benjamin Disraeli
+
+### The Earth Is Round (p < .05) [<sup>13</sup>](#Cohen1993)
+
+p值——以及更一般的，整个假设检验范式，在这次危机中起了核心作用。越来越多人知道这套范式不好，但大多数人还没意识到它到底有多糟，对此，备受尊敬的前美国心理学会会长保罗·米尔(Paul Meehl)是这么说的[<sup>14</sup>](#meehl1978theoretical)，
+
+> I believe that the almost universal reliance on merely refuting the null hypothesis as the standard method for corroborating substantive theories in the soft areas is a terrible mistake, is basically unsound, poor scientific strategy, and one of the worst things that ever happened in the history of psychology.
+
+p值带来了[太多的误解](https://en.wikipedia.org/wiki/Misunderstandings_of_p-values)[<sup>15</sup>](#greenland2016statistical)，连美国统计学会都少见的出来说了几句[<sup>16</sup>](#wasserstein2016asa)，这里是几点主要的。
```

Comment:
**P值带来了太多的误解** 这句话不对，wiki 和参考文献15表达的意思是很多人误解P值的含义，这个锅应该是这些人来背，而不是P值来背。参考文献15的 **Abstract** 说的很清楚： **Misinterpretation and abuse of statistical tests, confidence intervals, and statistical power have been decried for decades, yet remain rampant.** （开头第一句定下基调），**Our goal is to provide a resource for instructors, researchers, and consumers of statistics whose knowledge of statistical theory and technique may be limited but who wish to avoid and spot misinterpretations.** （后面定下文章目标、作用）